### PR TITLE
Api update

### PR DESCRIPTION
**Pull request by gabmor38**
Changes to the services and accessibility for the tiles.

### CODE REVIEW

File: src/services/GroupService.ts [hunk 1]
```diff
@@ -12,30 +12,74 @@ export class GroupServiceManager {
     this.context = context;
   }
 
-  public getGroups(letter: string): Promise<MicrosoftGraph.Group[]> {
+  // public getGroups(letter: string): Promise<MicrosoftGraph.Group[]> {
```
Comment (Steph4104) [inline src/services/GroupService.ts]: Just a small code clean up

File: src/webparts/reactAllGroups/components/reactAllGroups/ReactAllGroups.module.scss [hunk 2]
```diff
@@ -165,34 +169,34 @@
...
- transition: border-color 200ms;
-}
-
-/**
-* 1. Hide the default focus outline as it’s recreated with a border.
-*/
-.cardPrimaryAction:focus {
- outline: none; /* 1 */
-}
+// /** 1. Reset the default styles from a link.
```
Comment (Steph4104) [inline src/webparts/reactAllGroups/components/reactAllGroups/ReactAllGroups.module.scss]: Just a small code clean up

**Review by Steph4104 — APPROVED** [review]
All look good! But if you can do some code clean up before merge it will be nice.

File: src/services/GroupService.ts (hunk 1 above)
Comment (gabmor38) [reply src/services/GroupService.ts]: removed the commented out code

File: src/webparts/reactAllGroups/components/reactAllGroups/ReactAllGroups.module.scss (hunk 2 above)
Comment (gabmor38) [reply src/webparts/reactAllGroups/components/reactAllGroups/ReactAllGroups.module.scss]: removed the commented out code

**Pull request merged**